core/map2.f

```diff
@@ -397,15 +397,24 @@ subroutine read_con(wk,nwk,nelr,nv,ierr)
             call byte_read(hdr,sizeof(hdr)/4,ierr)
             if(ierr.ne.0) goto 100
 
-            read (hdr,*) version,nelgti,nelgvi,nvi
-c    1       format(a5,2i12,i2)
+            read (hdr,'(a5)') version 
+
+            if (version.eq.'#v002') then
```

Comment:
Why do we need a new header version `#v002` if the old one was using free format already?

Reply:
All co2 #v001 using i12 generated from existing tools will not get affected anyway.

In n2to3co3: we still want to support co2 generation with #v001, but for E<10M. We set E=10M as the transition criteria to big size (10M is to have more users test out). For E>=10M, the header uses #v002 with i16.  re2 generation (#v004) is based on the same approach, supporting #v002 and #v004 using E=10M as the separation point.

And so the decision #v002 for co2 was for the consistency with re2 using i16 and E=10M criteria, indicating the jump to 1B support. 

Theoretically, things will have to work up to 2B. More than a dozen tests were done with 1B level. The range of 1B-2B has to be done more testings to claim 2B support with additional changes at next round.